### ISSUE TITLE
How to iframe

### ISSUE THREAD

**Issue by korzq** [Zarel/Pokemon-Showdown#5215]
Hello! How do I iframe the index.html in static so it does't redirect to psim.us in the client side?

**Comment by xfix** [comment]
Could you clarify what you mean by that? What have you tried? I'm trying to understand what you are saying, and still didn't.

**Comment by Zarel** [comment]
I actually don't think PS even has any iframe-killers at the moment.

**Comment by korzq** [comment]
Sorry I did not make myself clear. When I start my PS Server, when I connect to the domain name, it redirects to a psim.us because of the `document.location.replace` in Pokemon-Showdown/server/static/index.html 
I would like to hide the domain to only show mine server's domain(pokemon.example.com) instead of (pokemon.example.com-80.psim.us). I know that is possible from line 76 of README.md

I tried
* add the <iframe> tag outside of <script> 
* iframe function in js `var ifrm = document.createElement("iframe");` then replacing `document.location.replace` with `ifrm.document.location.replace`
But none of them worked. (sorry I am really bad at this but I am trying my best)
